README.md

```diff
@@ -18,6 +18,16 @@ helm repo add fluent https://fluent.github.io/helm-charts
 
 You can then run `helm search repo fluent` to see the charts.
 
+## Upgrading
```

Comment:
This section does not belong here.

Please move it to `charts/fluentd/README.md` and update other respective section where you see fit